<?xml version="1.0"?>
<launch>
    <!-- <master auto="start"/>  -->
    <param name="param_yaml_file" value="$(find xnrobot_id11)/param/znnz2024.yaml" />

    <!-- Run the map server -->
    <node name="map_server" pkg="map_server" type="map_server" args="$(find xnrobot_id11)/map/map.yaml"/>

    <node pkg="xnrobot_id11" type="tf_manager.py" name="tf_manager">
    </node>

    <node pkg="xnrobot_id11" type="laser_filter.py" name="laser_filter">
    </node>

    <!--      als -->
    <include file="$(find als_ros)/launch/mcl_xnrobot_id11.launch" />

    

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find xnrobot_id11)/rviz_cfg/znnz2024_sim.rviz"/>
</launch> 

